packages/blade/src/components/Button/BaseButton/getStyledBaseButtonStyles.ts

```diff
@@ -29,6 +29,10 @@ const getBaseButtonStyles = ({
   paddingBottom: buttonPaddingBottom,
   paddingLeft: buttonPaddingLeft,
   paddingRight: buttonPaddingRight,
+  display: 'inline-flex',
+  justifyContent: 'center',
+  alignItems: 'center',
```

Comment:
Just check once if full width buttons are working okay on native. I recall facing some issues with full width when we give align and justify properties but I think that was with alignSelf

Reply:
<img width="320" alt="image" src="https://github.com/razorpay/blade/assets/30949385/8e35c7fc-8f12-4bb6-9743-3cb59ed4e8d9">

Seems to look good.